tests/ui/rfcs/rfc-2396-target_feature-11/trait-impl.stderr

```diff
@@ -1,3 +1,17 @@
+error[E0053]: method `foo` has an incompatible type for trait
+  --> $DIR/trait-impl.rs:15:5
+   |
+LL |     fn foo(&self) {}
+   |     ^^^^^^^^^^^^^ expected safe fn, found unsafe fn
+   |
+note: type in trait
+  --> $DIR/trait-impl.rs:6:5
+   |
+LL |     fn foo(&self);
+   |     ^^^^^^^^^^^^^^
+   = note: expected signature `fn(&Bar)`
+              found signature `unsafe fn(&Bar)`
```

Comment:
this is still wrong... why are we repeating the same logic in ~~50~~ 3 places... go fix this

Reply:
Ah this is pure fndef printing.. hmm